src/public_pool/Dockerfile

```diff
@@ -10,13 +10,8 @@ FROM node:18.19.1-bullseye-slim AS build
 RUN apt-get update \
     && apt-get upgrade -y
 
-# extra installs necessary for ARM computers to be able to compile
-RUN apt-get install -y --no-install-recommends python3
-RUN apt-get install -y --no-install-recommends build-essential 
-RUN apt-get install -y --no-install-recommends cmake g++ make
-RUN apt-get install -y --no-install-recommends ca-certificates 
-RUN apt-get install -y --no-install-recommends curl 
-RUN apt-get install -y --no-install-recommends procps
+# Extra installs necessary for ARM computers to be able to compile
+RUN apt-get install -y --no-install-recommends python3 build-essential cmake g++ make ca-certificates curl procps
```

Comment:
If one of these fails to install the entire docker build fails. If they are on their own separate lines, it's easier to see which one causes the failure. Because Parmanode runs on many different environments, and applications keep updating, it's not certain that something that installs today will continue to work tomorrow.

This is the advantage of keeping separate lines.
The benefit to putting them all on one line seem small in comparison.